web_company_color/models/res_company.py

```diff
@@ -106,12 +106,17 @@ def write(self, values):
     @api.multi
     def _scss_get_sanitized_values(self):
         self.ensure_one()
+        # Clone company_color as dictionary to avoid ORM operations
+        # This allow extend company_colors and only sanitize selected fields
+        # or add custom values
         values = dict(self.company_colors)
```

Comment:
this could be more safe like `dict(self.company_colors or {})`

Reply:
Done!